### PR TITLE
Add firstPatchedVersion in github API

### PR DESCRIPTION
**Pull request by TG1999**
The GitHub importer failed to parse Composer packages who did not have a slash in their names. We now no longer raise an exception if a Composer package does not have "vendor/name" but only a plain name.

Also we were not getting the first patched version from the GitHub API using GraphQL. This PR is also fixing this.

Reference: https://github.com/nexb/vulnerablecode/issues/645
Signed-off-by: Tushar Goel <tushar.goel.dav@gmail.com>

### CODE REVIEW

File: vulnerabilities/importers/github.py [hunk 1]
```diff
@@ -310,12 +319,8 @@ def process_response(resp: dict, package_type: str) -> Iterable[AdvisoryData]:
             else:
                 logger.error(f"Unknown identifier type {identifier_type!r} and value {value!r}")
 
-        date_published = get_item(advisory, "publishedAt")
-        if date_published:
-            date_published = dateparser.parse(date_published)
-
         yield AdvisoryData(
-            aliases=sorted(list(aliases)),
```
Comment (pombredanne) [inline vulnerabilities/importers/github.py:318]: IMHO it is important to get sorting back in there.

**Review by pombredanne — APPROVED** [review]
LGTM... Please double check the get item() , may be to ensure you have corner cases testsed.

**Comment by TG1999** [comment]
Thanks! added tests for the corner cases, merging now!
